### DISCUSSION TITLE
XSD not importing as data map into Excel

### DISCUSSION THREAD

**Discussion by cosmicv** [usnistgov/OSCAL#1858] [Q&A] [answered]
I've tried 3 different computers and every time I try to import oscal_ssp_schema.xsd into Excel as a data map, it simply hangs. I verified the XSD with various tools. Am I missing something? Thanks!

**Comment by iMichaela** [comment, marked as answer]
The oscal_ssp_schema.xsd is the schema used to validate OSCAL SSP content (an SSP file in OSCAL XML) and it contains several types of XML schema element constructs Excel doesn't support.
